### PR TITLE
[zh] Sync ambient warning with #14405 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync ambient warning with #14405 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ Kubernetes 版本 ({{< supported_kubernetes_versions >}})。
 您可以使用 [Minikube](https://kubernetes.io/zh-cn/docs/tasks/tools/install-minikube/)
 或[特定平台搭建指南](/zh/docs/setup/platform-setup/)中指定的所有受支持的平台。
 
+{{< warning >}}
+请注意，Ambient 目前需要使用 [istio-cni](/zh/docs/setup/additional-setup/cni)
+来配置 Kubernetes 节点。`istio-cni` Ambient
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/getting-started/index.md:20]: nit
```suggestion
来配置 Kubernetes 节点。`istio-cni` 的 Ambient
```

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 2]
```diff
@@ -15,6 +15,13 @@ Kubernetes 版本 ({{< supported_kubernetes_versions >}})。
...
+模式当前**不可以**支持集群 CNI 类型（即不可以使用 `veth` 设备的 CNI 实现，
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/getting-started/index.md:21]:
```suggestion
模式当前**不可以**支持某些类型的集群 CNI（如不使用 `veth` 设备的 CNI 实现，
```

**Review by windsonsea — COMMENTED** [review]
See [preview](https://deploy-preview-14407--preliminary-istio.netlify.app/latest/zh/docs/ops/ambient/getting-started/).

**Review by windsonsea — APPROVED** [review]
lgtm, thanks.

**Pull request merged**